### PR TITLE
Tomi frontend dev

### LINKED ISSUE #13: Creating front-end of site
Goal is to create page appearance as similar to pottershop.pl's as possible. It will probably require to work with pure code in /potershop directory. Remember to change all default values defined by prestashop creator (name of the website etc.). Interface MUST be in polish

### PR DESCRIPTION
**Pull request by tomaszsankowski**
Almost complete front end of application. Only thing left is to add product slider on product site and small changes to make shop even more simmilar to copied one.

### CODE REVIEW

File: api/.env
Comment (peterprospl12) [inline api/.env:1]: delete this file

File: api/__pycache__/prestashopAPI.cpython-312.pyc
Comment (peterprospl12) [inline api/__pycache__/prestashopAPI.cpython-312.pyc:1]: delete all pycache dir

**Review by peterprospl12 — CHANGES_REQUESTED** [review]

File: api/__pycache__/prestashopAPI.cpython-312.pyc
Comment (tomaszsankowski) [reply api/__pycache__/prestashopAPI.cpython-312.pyc:1]: Done in the latest commit. Thanks

File: api/.env
Comment (tomaszsankowski) [reply api/.env:1]: Done in the latest commit. Thanks

Comment (peterprospl12) [reply api/.env:1]: no problem man, i'm here to help

File: api/__pycache__/prestashopAPI.cpython-312.pyc
Comment (peterprospl12) [reply api/__pycache__/prestashopAPI.cpython-312.pyc:1]: no problem man, i'm here to help

File: prestashop/themes/child_classic/modules/ps_searchbar/ps_searchbar.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    public function hookDisplayHeader()
+    {
+        $this->context->controller->addJqueryUI('ui.autocomplete');
+        $this->context->controller->registerStylesheet('modules-searchbar', 'modules/' . $this->name . '/ps_searchbar.css');
+        $this->context->controller->registerJavascript('modules-searchbar', 'modules/' . $this->name . '/ps_searchbar.js', ['position' => 'bottom', 'priority' => 150]);
+    }
+
+    public function getWidgetVariables($hookName, array $configuration = [])
+    {
+        $widgetVariables = [
```
Comment (peterprospl12) [inline prestashop/themes/child_classic/modules/ps_searchbar/ps_searchbar.php:90]: good job i like it

File: prestashop/themes/child_classic/modules/ps_shoppingcart/ps_shoppingcart-product-line.tpl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<span class="product-price">{$product.price}</span>
+<a  class="remove-from-cart"
+    rel="nofollow"
+    href="{$product.remove_from_cart_url}"
+    data-link-action="remove-from-cart"
+    title="{l s='remove from cart' d='Shop.Theme.Actions'}"
+>
+    {l s='Remove' d='Shop.Theme.Actions'}
+</a>
+{if $product.customizations|count}
```
Comment (peterprospl12) [inline prestashop/themes/child_classic/modules/ps_shoppingcart/ps_shoppingcart-product-line.tpl:36]: wow!

File: prestashop/webservice/yaml/76a40f46af300c63b18d3b29c5a7337b.php [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+        'height' => 98,
+        'scope' => 
+        array (
+          0 => 'products',
+          1 => 'categories',
+          2 => 'manufacturers',
+          3 => 'suppliers',
+        ),
+      ),
+      'medium_default' =>
```
Comment (peterprospl12) [inline prestashop/webservice/yaml/76a40f46af300c63b18d3b29c5a7337b.php:174]: good job

**Review by peterprospl12 — APPROVED** [review]
good job man, LGTM

**Pull request merged**